spec/requests/supervisor_volunteers_spec.rb

```diff
@@ -102,7 +101,7 @@
         }.not_to change(supervisor.volunteers, :count)
 
         association.reload
-
+        
```

Comment:
I recommend configuring VSCode to auto-trim trailing spaces on save - probably something like this https://stackoverflow.com/questions/30884131/remove-trailing-spaces-automatically-or-with-a-shortcut